app/controllers/adjustments_controller.rb

```diff
@@ -1,26 +1,33 @@
 class AdjustmentsController < ApplicationController
   before_action :set_adjustment, only: %i[ show edit update destroy ]
+  after_action :verify_policy_scoped, only: [:index]
+  after_action :verify_authorized, except: [:index]
 
   # GET /adjustments or /adjustments.json
   def index
-    @adjustments = current_user.adjustments.all
+    #@adjustments = current_user.adjustments.all
+    @adjustments = policy_scope(Adjustment)
   end
 
   # GET /adjustments/1 or /adjustments/1.json
   def show
+    authorize @adjustment
   end
 
   # GET /adjustments/new
   def new
+    authorize Adjustment
     @adjustment = Adjustment.new
   end
 
   # GET /adjustments/1/edit
   def edit
+    authorize @adjustment
   end
 
   # POST /adjustments or /adjustments.json
   def create
+    authorize Adjustment
```

Comment:
You can authorize the new record should have all the initial columns set, except for `id`, `created_at`, `updated_at`. So if you needed to restrict based on user or role in the policy you could.
```suggestion
    @adjustment = Adjustment.new(adjustment_params)
    authorize @adjustment
```

Reply:
This makes sense.  I will add this to both the `loan` and `adjustment` controller. Thanks!